Formula/jfrog-cli.rb

```diff
@@ -17,14 +17,17 @@ class JfrogCli < Formula
   depends_on "go" => :build
 
   def install
-    system "go", "build", "-ldflags", "-s -w -extldflags '-static'", "-trimpath", "-o", bin/"jfrog"
+    system "go", "build", "-ldflags", "-s -w -extldflags '-static'", "-trimpath", "-o", bin/"jf"
```

Comment:
While we're changing this, let's use the newer syntax for this:
```suggestion
    system "go", "build", *std_go_args(ldflags: "-s -w -extldflags '-static'", output: bin/"jf")
```